dataset_builders/pie/abstrct/README.md

```diff
@@ -92,22 +105,236 @@ Morio et al. ([2022](https://aclanthology.org/2022.tacl-1.37.pdf); p. 642, Table
 
 (Mayer et al. 2020, p.2110)
 
-#### Examples
+#### Example
 
-![Examples](img/abstr-sam.png)
+![abstr-sam.png](img%2Fabstr-sam.png)
 
-### Document Converters
+### Collected Statistics after Document Conversion
 
-The dataset provides document converters for the following target document types:
+In this section, we collect further statistics of the dataset after the conversion to `TextDocumentWithLabeledSpansAndBinaryRelations`.
+In the commands, we used the following dataset config: `configs/dataset/abstrct_base.yaml`:
```

Comment:
First, give a small description how you got the statistics / histograms at all, e.g.:


We use the code from [PyTorch-IE-Hydra-Template](https://github.com/ArneBinder/pytorch-ie-hydra-template-1) to ... After checking out that code, the statistics and plots can be generated by the command:
```bash
python src/evaluate_documents.py dataset=abstrct_base metric=METRIC
```
where METRIC is ...

This also requires to have the following dataset config in `configs/dataset/abstrct_base.yaml` of within the repo directory:
```yaml
as before
```
For token based metrics, this uses `bert-base-uncased` from `transformer.AutoTokenizer` (see [AutoTokenizer](https://huggingface.co/docs/transformers/v4.37.1/en/model_doc/auto#transformers.AutoTokenizer), and [bert-based-uncased](https://huggingface.co/bert-base-uncased)) to tokenize `text` in `TextDocumentWithLabeledSpansAndBinaryRelations` (see [document type](https://github.com/ChristophAlt/pytorch-ie/blob/main/src/pytorch_ie/documents.py)).